### ISSUE TITLE
tests broken on macos with julia 1.3

### ISSUE THREAD

**Issue by AshtonSBradley** [JuliaDiffEq/DifferentialEquations.jl#533]
There seems to be a problem on MacOS. 

I thought maybe I had corrupted something, so in the end I blew away `.julia`.

On a completely fresh install:
```(v1.3) pkg> test DifferentialEquations
   Testing DifferentialEquations
 Resolving package versions...
 Installed SafeTestsets ──────────── v0.0.1
 Installed LaTeXStrings ──────────── v1.0.3
 Installed DynamicPolynomials ────── v0.1.4
 Installed ProgressMeter ─────────── v0.9.0
 Installed DiffEqBiological ──────── v4.0.2
 Installed DiffEqProblemLibrary ──── v4.3.0
 Installed DiffEqOperators ───────── v4.1.0
 Installed HomotopyContinuation ──── v0.7.0
 Installed TimerOutputs ──────────── v0.5.3
 Installed MultivariatePolynomials ─ v0.3.2
 Installed ParameterizedFunctions ── v4.2.1
 Installed FixedPolynomials ──────── v0.4.0
 Installed StaticPolynomials ─────── v1.3.3
 Installed NNlib ─────────────────── v0.6.0
 Installed SymEngine ─────────────── v0.7.0
 Installed ProjectiveVectors ─────── v0.2.2
 Installed Latexify ──────────────── v0.12.3
  Building SymEngine → `~/.julia/packages/SymEngine/zSUGO/deps/build.log`
    Status `/var/folders/3g/gsxf8njs27z_2ql1xk1gb7hw0000gn/T/jl_eenl4y/Manifest.toml`
  [79e6a3ab] Adapt v1.0.0
  [ec485272] ArnoldiMethod v0.0.4
  [4fba245c] ArrayInterface v1.2.1
  [4c555306] ArrayLayouts v0.1.5
  [aae01518] BandedMatrices v0.14.2
  [b99e7846] BinaryProvider v0.5.8
  [764a87c0] BoundaryValueDiffEq v2.3.0
  [49dc2e85] Calculus v0.5.1
  [bbf7d656] CommonSubexpressions v0.2.0
  [34da2185] Compat v2.2.0
  [864edb3b] DataStructures v0.17.6
  [bcd4f6db] DelayDiffEq v5.18.0
  [2b5f629d] DiffEqBase v6.7.0
  [eb300fae] DiffEqBiological v4.0.2
  [459566f4] DiffEqCallbacks v2.9.0
  [01453d9d] DiffEqDiffTools v1.4.0
  [5a0ffddc] DiffEqFinancial v2.2.1
  [c894b116] DiffEqJump v6.3.0
  [77a26b50] DiffEqNoiseProcess v3.5.0
  [9fdde737] DiffEqOperators v4.1.0
  [055956cb] DiffEqPhysics v3.2.0
  [a077e3f3] DiffEqProblemLibrary v4.3.0
  [163ba53b] DiffResults v0.0.4
  [b552c78f] DiffRules v0.1.0
  [0c46a032] DifferentialEquations v6.8.0
  [c619ae07] DimensionalPlotRecipes v1.0.0
  [b4f34e82] Distances v0.8.2
  [ffbed154] DocStringExtensions v0.8.1
  [7c1d4256] DynamicPolynomials v0.1.4
  [d4d017d3] ExponentialUtilities v1.5.1
  [1a297f60] FillArrays v0.8.2
  [3dd14ad9] FixedPolynomials v0.4.0
  [f6369f11] ForwardDiff v0.10.7
  [069b7b12] FunctionWrappers v1.0.0
  [01680d73] GenericSVD v0.2.2
  [f213a82b] HomotopyContinuation v0.7.0
  [d25df0c9] Inflate v0.1.1
  [42fd0dbc] IterativeSolvers v0.8.1
  [82899510] IteratorInterfaceExtensions v1.0.0
  [b964fa9f] LaTeXStrings v1.0.3
  [23fbe1c1] Latexify v0.12.3
  [093fc24a] LightGraphs v1.3.0
  [d3d80556] LineSearches v7.0.1
  [1914dd2f] MacroTools v0.5.2
  [46d2c3a1] MuladdMacro v0.2.1
  [f9640e96] MultiScaleArrays v1.5.0
  [102ac46a] MultivariatePolynomials v0.3.2
  [d41bc354] NLSolversBase v7.5.0
  [2774e3e8] NLsolve v4.2.0
  [872c559c] NNlib v0.6.0
  [77ba4419] NaNMath v0.3.3
  [efe28fd5] OpenSpecFun_jll v0.5.3+1
  [bac558e1] OrderedCollections v1.1.0
  [1dea7af3] OrdinaryDiffEq v5.24.0
  [65888b18] ParameterizedFunctions v4.2.1
  [d96e819e] Parameters v0.12.0
  [e409e4f3] PoissonRandom v0.4.0
  [92933f4c] ProgressMeter v0.9.0
  [01f381cc] ProjectiveVectors v0.2.2
  [e6cf234a] RandomNumbers v1.3.0
  [3cdcf5f2] RecipesBase v0.7.0
  [731186ca] RecursiveArrayTools v1.0.2
  [f2c3362d] RecursiveFactorization v0.1.0
  [189a3867] Reexport v0.2.0
  [ae029012] Requires v0.5.2
  [ae5879a3] ResettableStacks v0.6.0
  [f2b01f46] Roots v0.8.3
  [1bc83da4] SafeTestsets v0.0.1
  [699a6c99] SimpleTraits v0.9.1
  [47a9eef4] SparseDiffTools v1.0.0
  [276daf66] SpecialFunctions v0.9.0
  [90137ffa] StaticArrays v0.12.1
  [62e018b1] StaticPolynomials v1.3.3
  [9672c7b4] SteadyStateDiffEq v1.5.0
  [789caeaf] StochasticDiffEq v6.15.0
  [c3572dad] Sundials v3.8.0
  [123dc426] SymEngine v0.7.0
  [3783bdb8] TableTraits v1.0.0
  [a759f4b9] TimerOutputs v0.5.3
  [a2a6695c] TreeViews v0.3.0
  [19fa3120] VertexSafeGraphs v0.1.0
  [2a0f44e3] Base64  [`@stdlib/Base64`]
  [ade2ca70] Dates  [`@stdlib/Dates`]
  [8bb1440f] DelimitedFiles  [`@stdlib/DelimitedFiles`]
  [8ba89e20] Distributed  [`@stdlib/Distributed`]
  [b77e0a4c] InteractiveUtils  [`@stdlib/InteractiveUtils`]
  [76f85450] LibGit2  [`@stdlib/LibGit2`]
  [8f399da3] Libdl  [`@stdlib/Libdl`]
  [37e2e46d] LinearAlgebra  [`@stdlib/LinearAlgebra`]
  [56ddb016] Logging  [`@stdlib/Logging`]
  [d6f4376e] Markdown  [`@stdlib/Markdown`]
  [a63ad114] Mmap  [`@stdlib/Mmap`]
  [44cfe95a] Pkg  [`@stdlib/Pkg`]
  [de0858da] Printf  [`@stdlib/Printf`]
  [3fa0cd96] REPL  [`@stdlib/REPL`]
  [9a3f8284] Random  [`@stdlib/Random`]
  [ea8e919c] SHA  [`@stdlib/SHA`]
  [9e88b42a] Serialization  [`@stdlib/Serialization`]
  [1a1011a3] SharedArrays  [`@stdlib/SharedArrays`]
  [6462fe0b] Sockets  [`@stdlib/Sockets`]
  [2f01184e] SparseArrays  [`@stdlib/SparseArrays`]
  [10745b16] Statistics  [`@stdlib/Statistics`]
  [4607b0f0] SuiteSparse  [`@stdlib/SuiteSparse`]
  [8dfed614] Test  [`@stdlib/Test`]
  [cf7118a7] UUIDs  [`@stdlib/UUIDs`]
  [4ec0a83e] Unicode  [`@stdlib/Unicode`]
Starting tests
Test Summary:              | Pass  Total
Default Discrete Algorithm |    1      1
  4.279644 seconds (13.87 M allocations: 712.977 MiB, 8.44% gc time)
Default ODE Algorithm: Error During Test at /Users/abradley/.julia/packages/SafeTestsets/A83XK/src/SafeTestsets.jl:25
  Got exception outside of a @test
  LoadError: LoadError: LoadError: MethodError: no method matching DiffEqOperators.DerivativeOperator{Float64,N,Wind,T2,S1,S2,T3,F} where F where T3 where S2<:(StaticArrays.SArray{Tuple{S},T,1,S} where T where S) where S1 where T2 where Wind where N(::Int64, ::Int64, ::Float64, ::Int64, ::Symbol, ::Symbol; BC=(1.0, 1.0))
  Stacktrace:
   [1] top-level scope at /Users/abradley/.julia/packages/DiffEqProblemLibrary/3xoHQ/src/ode/brusselator_prob.jl:100
   [2] include at ./boot.jl:328 [inlined]
   [3] include_relative(::Module, ::String) at ./loading.jl:1105
   [4] include at ./Base.jl:31 [inlined]
   [5] include(::String) at /Users/abradley/.julia/packages/DiffEqProblemLibrary/3xoHQ/src/DiffEqProblemLibrary.jl:3
   [6] top-level scope at /Users/abradley/.julia/packages/DiffEqProblemLibrary/3xoHQ/src/ode/ode_premade_problems.jl:18
   [7] include at ./boot.jl:328 [inlined]
   [8] include_relative(::Module, ::String) at ./loading.jl:1105
   [9] include at ./Base.jl:31 [inlined]
   [10] include at /Users/abradley/.julia/packages/DiffEqProblemLibrary/3xoHQ/src/DiffEqProblemLibrary.jl:3 [inlined]
   [11] importodeproblems() at /Users/abradley/.julia/packages/DiffEqProblemLibrary/3xoHQ/src/DiffEqProblemLibrary.jl:5
   [12] top-level scope at /Users/abradley/.julia/packages/DifferentialEquations/uUFZS/test/default_ode_alg_test.jl:3
   [13] include at ./boot.jl:328 [inlined]
   [14] include_relative(::Module, ::String) at ./loading.jl:1105
   [15] include at ./Base.jl:31 [inlined]
   [16] include(::String) at /Users/abradley/.julia/packages/SafeTestsets/A83XK/src/SafeTestsets.jl:23
   [17] top-level scope at /Users/abradley/.julia/packages/DifferentialEquations/uUFZS/test/runtests.jl:6
   [18] top-level scope at /Users/sabae/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.3/Test/src/Test.jl:1107
   [19] top-level scope at /Users/abradley/.julia/packages/DifferentialEquations/uUFZS/test/runtests.jl:6
   [20] eval(::Module, ::Any) at ./boot.jl:330
   [21] top-level scope at /Users/abradley/.julia/packages/SafeTestsets/A83XK/src/SafeTestsets.jl:23
   [22] top-level scope at util.jl:155
   [23] include at ./boot.jl:328 [inlined]
   [24] include_relative(::Module, ::String) at ./loading.jl:1105
   [25] include(::Module, ::String) at ./Base.jl:31
   [26] include(::String) at ./client.jl:424
   [27] top-level scope at none:6
   [28] eval(::Module, ::Any) at ./boot.jl:330
   [29] exec_options(::Base.JLOptions) at ./client.jl:263
   [30] _start() at ./client.jl:460
  in expression starting at /Users/abradley/.julia/packages/DiffEqProblemLibrary/3xoHQ/src/ode/brusselator_prob.jl:100
  in expression starting at /Users/abradley/.julia/packages/DiffEqProblemLibrary/3xoHQ/src/ode/ode_premade_problems.jl:18
  in expression starting at /Users/abradley/.julia/packages/DifferentialEquations/uUFZS/test/default_ode_alg_test.jl:3
  
Test Summary:         | Error  Total
Default ODE Algorithm |     1      1
ERROR: LoadError: Some tests did not pass: 0 passed, 0 failed, 1 errored, 0 broken.
in expression starting at /Users/abradley/.julia/packages/DifferentialEquations/uUFZS/test/runtests.jl:6
ERROR: Package DifferentialEquations errored during testing```

with

```
julia> versioninfo()
Julia Version 1.3.0
Commit 46ce4d7933 (2019-11-26 06:09 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin19.0.0)
  CPU: Intel(R) Core(TM) i5-8259U CPU @ 2.30GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, skylake)
Environment:
  JULIA_NUM_THREADS = 4
```

**Comment by ChrisRackauckas** [comment]
DiffEqProblemLibrary is a very old version. Can you find out why?

**Comment by AshtonSBradley** [comment]
How do I find out?

**Comment by AshtonSBradley** [comment]
I haven't done anything else except `]add DifferentialEquations` on a fresh install of julia...

**Comment by ChrisRackauckas** [comment]
Try to directly add DiffEqProblemLibrary 4.5.1 and see what it says.

**Comment by AshtonSBradley** [comment]
ok so that installed just fine, no complaints, but StaticArrays got downgraded:
```juila 
(v1.3) pkg> add DiffEqProblemLibrary@4.5.1
  Updating registry at `~/.julia/registries/General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
 Resolving package versions...
 Installed Tables ─────────────── v0.2.11
 Installed DiffEqProblemLibrary ─ v4.5.1
 Installed MixedSubdivisions ──── v0.3.4
 Installed ProjectiveVectors ──── v1.1.2
 Installed StaticArrays ───────── v0.11.1
 Installed ModelingToolkit ────── v0.10.0
 Installed DoubleFloats ───────── v0.9.13
 Installed DataValueInterfaces ── v1.0.0
 Installed ElasticArrays ──────── v1.0.0
 Installed Formatting ─────────── v0.4.1
 Installed JuliaVariables ─────── v0.2.0
 Installed GeneralizedGenerated ─ v0.2.0
 Installed LazyArrays ─────────── v0.14.10
 Installed NameResolution ─────── v0.1.3
 Installed DiffEqOperators ────── v4.5.0
 Installed MatrixFactorizations ─ v0.2.1
 Installed BlockArrays ────────── v0.10.2
 Installed PrettyPrint ────────── v0.1.0
 Installed DynamicPolynomials ─── v0.3.3
 Installed Quadmath ───────────── v0.5.2
 Installed MLStyle ────────────── v0.3.1
 Installed Crayons ────────────── v4.0.1
 Installed HomotopyContinuation ─ v1.2.1
 Installed BlockBandedMatrices ── v0.6.0
 Installed CanonicalTraits ────── v0.1.0
 Installed GenericSchur ───────── v0.3.0
 Installed PrettyTables ───────── v0.6.0
  Updating `~/.julia/environments/v1.3/Project.toml`
  [a077e3f3] + DiffEqProblemLibrary v4.5.1
  Updating `~/.julia/environments/v1.3/Manifest.toml`
  [8e7c35d0] + BlockArrays v0.10.2
  [ffab5731] + BlockBandedMatrices v0.6.0
  [a603d957] + CanonicalTraits v0.1.0
  [a8cc5b0e] + Crayons v4.0.1
  [e2d170a0] + DataValueInterfaces v1.0.0
  [eb300fae] + DiffEqBiological v4.0.2
  [9fdde737] + DiffEqOperators v4.5.0
  [a077e3f3] + DiffEqProblemLibrary v4.5.1
  [497a8b3b] + DoubleFloats v0.9.13
  [7c1d4256] + DynamicPolynomials v0.3.3
  [fdbdab4c] + ElasticArrays v1.0.0
  [3dd14ad9] + FixedPolynomials v0.4.0
  [59287772] + Formatting v0.4.1
  [6b9d7cbe] + GeneralizedGenerated v0.2.0
  [c145ed77] + GenericSchur v0.3.0
  [f213a82b] + HomotopyContinuation v1.2.1
  [b14d175d] + JuliaVariables v0.2.0
  [b964fa9f] + LaTeXStrings v1.0.3
  [23fbe1c1] + Latexify v0.12.3
  [5078a376] + LazyArrays v0.14.10
  [d8e11817] + MLStyle v0.3.1
  [a3b82374] + MatrixFactorizations v0.2.1
  [291d046c] + MixedSubdivisions v0.3.4
  [961ee093] + ModelingToolkit v0.10.0
  [102ac46a] + MultivariatePolynomials v0.3.2
  [872c559c] + NNlib v0.6.0
  [71a1bf82] + NameResolution v0.1.3
  [8162dcfd] + PrettyPrint v0.1.0
  [08abe8d2] + PrettyTables v0.6.0
  [01f381cc] + ProjectiveVectors v1.1.2
  [be4d8f0f] + Quadmath v0.5.2
  [90137ffa] ↓ StaticArrays v0.12.1 ⇒ v0.11.1
  [62e018b1] + StaticPolynomials v1.3.3
  [123dc426] + SymEngine v0.7.0
  [bd369af6] + Tables v0.2.11
  [a759f4b9] + TimerOutputs v0.5.3
```
Now tests seem to pass (with warnings):
```juila

(v1.3) pkg> test DifferentialEquations
   Testing DifferentialEquations
 Resolving package versions...
    Status `/var/folders/3g/gsxf8njs27z_2ql1xk1gb7hw0000gn/T/jl_cfngwz/Manifest.toml`
  [79e6a3ab] Adapt v1.0.0
  [ec485272] ArnoldiMethod v0.0.4
  [4fba245c] ArrayInterface v1.2.1
  [4c555306] ArrayLayouts v0.1.5
  [aae01518] BandedMatrices v0.14.2
  [9e28174c] BinDeps v0.8.10
  [b99e7846] BinaryProvider v0.5.8
  [8e7c35d0] BlockArrays v0.10.2
  [ffab5731] BlockBandedMatrices v0.6.0
  [764a87c0] BoundaryValueDiffEq v2.3.0
  [49dc2e85] Calculus v0.5.1
  [a603d957] CanonicalTraits v0.1.0
  [bbf7d656] CommonSubexpressions v0.2.0
  [34da2185] Compat v2.2.0
  [a8cc5b0e] Crayons v4.0.1
  [9a962f9c] DataAPI v1.1.0
  [864edb3b] DataStructures v0.17.6
  [e2d170a0] DataValueInterfaces v1.0.0
  [bcd4f6db] DelayDiffEq v5.18.0
  [2b5f629d] DiffEqBase v6.7.0
  [eb300fae] DiffEqBiological v4.0.2
  [459566f4] DiffEqCallbacks v2.9.0
  [01453d9d] DiffEqDiffTools v1.4.0
  [5a0ffddc] DiffEqFinancial v2.2.1
  [c894b116] DiffEqJump v6.3.0
  [77a26b50] DiffEqNoiseProcess v3.5.0
  [9fdde737] DiffEqOperators v4.5.0
  [055956cb] DiffEqPhysics v3.2.0
  [a077e3f3] DiffEqProblemLibrary v4.5.1
  [163ba53b] DiffResults v0.0.4
  [b552c78f] DiffRules v0.1.0
  [0c46a032] DifferentialEquations v6.8.0
  [c619ae07] DimensionalPlotRecipes v1.0.0
  [b4f34e82] Distances v0.8.2
  [ffbed154] DocStringExtensions v0.8.1
  [497a8b3b] DoubleFloats v0.9.13
  [7c1d4256] DynamicPolynomials v0.3.3
  [fdbdab4c] ElasticArrays v1.0.0
  [d4d017d3] ExponentialUtilities v1.5.1
  [1a297f60] FillArrays v0.8.2
  [3dd14ad9] FixedPolynomials v0.4.0
  [59287772] Formatting v0.4.1
  [f6369f11] ForwardDiff v0.10.7
  [069b7b12] FunctionWrappers v1.0.0
  [6b9d7cbe] GeneralizedGenerated v0.2.0
  [01680d73] GenericSVD v0.2.2
  [c145ed77] GenericSchur v0.3.0
  [f213a82b] HomotopyContinuation v1.2.1
  [d25df0c9] Inflate v0.1.1
  [42fd0dbc] IterativeSolvers v0.8.1
  [82899510] IteratorInterfaceExtensions v1.0.0
  [b14d175d] JuliaVariables v0.2.0
  [b964fa9f] LaTeXStrings v1.0.3
  [23fbe1c1] Latexify v0.12.3
  [5078a376] LazyArrays v0.14.10
  [093fc24a] LightGraphs v1.3.0
  [d3d80556] LineSearches v7.0.1
  [d8e11817] MLStyle v0.3.1
  [1914dd2f] MacroTools v0.5.2
  [a3b82374] MatrixFactorizations v0.2.1
  [291d046c] MixedSubdivisions v0.3.4
  [961ee093] ModelingToolkit v0.10.0
  [46d2c3a1] MuladdMacro v0.2.1
  [f9640e96] MultiScaleArrays v1.5.0
  [102ac46a] MultivariatePolynomials v0.3.2
  [d41bc354] NLSolversBase v7.5.0
  [2774e3e8] NLsolve v4.2.0
  [872c559c] NNlib v0.6.0
  [77ba4419] NaNMath v0.3.3
  [71a1bf82] NameResolution v0.1.3
  [bac558e1] OrderedCollections v1.1.0
  [1dea7af3] OrdinaryDiffEq v5.24.0
  [d96e819e] Parameters v0.12.0
  [e409e4f3] PoissonRandom v0.4.0
  [f27b6e38] Polynomials v0.6.0
  [8162dcfd] PrettyPrint v0.1.0
  [08abe8d2] PrettyTables v0.6.0
  [92933f4c] ProgressMeter v1.2.0
  [01f381cc] ProjectiveVectors v1.1.2
  [be4d8f0f] Quadmath v0.5.2
  [e6cf234a] RandomNumbers v1.3.0
  [3cdcf5f2] RecipesBase v0.7.0
  [731186ca] RecursiveArrayTools v1.0.2
  [f2c3362d] RecursiveFactorization v0.1.0
  [189a3867] Reexport v0.2.0
  [ae029012] Requires v0.5.2
  [ae5879a3] ResettableStacks v0.6.0
  [f2b01f46] Roots v0.8.3
  [1bc83da4] SafeTestsets v0.0.1
  [699a6c99] SimpleTraits v0.9.1
  [47a9eef4] SparseDiffTools v1.0.0
  [276daf66] SpecialFunctions v0.8.0
  [90137ffa] StaticArrays v0.11.1
  [62e018b1] StaticPolynomials v1.3.3
  [9672c7b4] SteadyStateDiffEq v1.5.0
  [789caeaf] StochasticDiffEq v6.15.0
  [c3572dad] Sundials v3.8.0
  [123dc426] SymEngine v0.7.0
  [3783bdb8] TableTraits v1.0.0
  [bd369af6] Tables v0.2.11
  [a759f4b9] TimerOutputs v0.5.3
  [a2a6695c] TreeViews v0.3.0
  [30578b45] URIParser v0.4.0
  [19fa3120] VertexSafeGraphs v0.1.0
  [2a0f44e3] Base64  [`@stdlib/Base64`]
  [ade2ca70] Dates  [`@stdlib/Dates`]
  [8bb1440f] DelimitedFiles  [`@stdlib/DelimitedFiles`]
  [8ba89e20] Distributed  [`@stdlib/Distributed`]
  [b77e0a4c] InteractiveUtils  [`@stdlib/InteractiveUtils`]
  [76f85450] LibGit2  [`@stdlib/LibGit2`]
  [8f399da3] Libdl  [`@stdlib/Libdl`]
  [37e2e46d] LinearAlgebra  [`@stdlib/LinearAlgebra`]
  [56ddb016] Logging  [`@stdlib/Logging`]
  [d6f4376e] Markdown  [`@stdlib/Markdown`]
  [a63ad114] Mmap  [`@stdlib/Mmap`]
  [44cfe95a] Pkg  [`@stdlib/Pkg`]
  [de0858da] Printf  [`@stdlib/Printf`]
  [3fa0cd96] REPL  [`@stdlib/REPL`]
  [9a3f8284] Random  [`@stdlib/Random`]
  [ea8e919c] SHA  [`@stdlib/SHA`]
  [9e88b42a] Serialization  [`@stdlib/Serialization`]
  [1a1011a3] SharedArrays  [`@stdlib/SharedArrays`]
  [6462fe0b] Sockets  [`@stdlib/Sockets`]
  [2f01184e] SparseArrays  [`@stdlib/SparseArrays`]
  [10745b16] Statistics  [`@stdlib/Statistics`]
  [4607b0f0] SuiteSparse  [`@stdlib/SuiteSparse`]
  [8dfed614] Test  [`@stdlib/Test`]
  [cf7118a7] UUIDs  [`@stdlib/UUIDs`]
  [4ec0a83e] Unicode  [`@stdlib/Unicode`]
Starting tests
Test Summary:              | Pass  Total
Default Discrete Algorithm |    1      1
  4.421929 seconds (13.27 M allocations: 685.609 MiB, 7.78% gc time)
┌ Warning: Replacing docs for `DiffEqOperators.PeriodicBC :: Union{}` in module `DiffEqOperators`
└ @ Base.Docs docs/Docs.jl:223
┌ Warning: Replacing docs for `DiffEqOperators.RobinBC :: Union{}` in module `DiffEqOperators`
└ @ Base.Docs docs/Docs.jl:223
Test Summary:         | Pass  Total
Default ODE Algorithm |   21     21
163.809969 seconds (341.26 M allocations: 15.396 GiB, 3.31% gc time)
Test Summary:                  | Pass  Total
Default Steady State Algorithm |    1      1
  6.248019 seconds (23.03 M allocations: 1.167 GiB, 5.35% gc time)
Test Summary:         | Pass  Total
Default SDE Algorithm |    8      8
 94.942102 seconds (168.37 M allocations: 35.026 GiB, 7.26% gc time)
Test Summary:          | Pass  Total
Default RODE Algorithm |    1      1
  8.464831 seconds (22.61 M allocations: 6.949 GiB, 15.75% gc time)
Test Summary:         | Pass  Total
Default DDE Algorithm |    3      3
  5.566602 seconds (14.90 M allocations: 792.695 MiB, 4.02% gc time)
Test Summary:         | Pass  Total
Default DAE Algorithm |    1      1
  2.073117 seconds (5.34 M allocations: 282.466 MiB, 2.81% gc time)
   Testing DifferentialEquations tests passed 
```

**Comment by ChrisRackauckas** [comment]
None of JuliaDiffEq bounds StaticArrays 0.12, but it is a very very new major release and so it's possible something (or many things) outside of JuliaDiffEq still bound it. So I'm not surprised you need 0.11 or something of the likes to run the tests. That said, there's nothing actionable here, and it doesn't impact general usage (the fact that it cannot downgrade at test time is a Pkg bug that's fixed in 1.4 IIRC), so I'm closing.
